### DISCUSSION TITLE
naive组件可以结合UnoCSS图标一起使用吗？

### DISCUSSION THREAD

**Discussion by ShangChien** [tusen-ai/naive-ui#2964] [Q&A] [answered]
在segmentfault上看到了[一片帖子](https://segmentfault.com/a/1190000041792195###)，介绍怎么在UI组件中使用UnoCSS图标（案例并不是Naive UI）。实用主义新手，不太懂原子化这类东西。直接使用div包裹的UnoCSS图标挺方便的（其实Naive UI提供的图标方案也很方便）。单纯好奇想知道在Naive组件中，如<n-input/>的前缀后缀，有没有直接使用UnoCSS图标的方法？

**Comment by 07akioni** [comment, marked as answer]
这两者没啥不能结合的吧？获取 icon 放进 `n-icon` 就好了

**Reply by ShangChien** [reply]
我刚刚简单试了一下，生成的图标好小一个，不能改大小，可能是我的安装配置的方式有差，有空再捣鼓一下。

**Reply by 07akioni** [reply]
有问题的话可以提个复现
